resources/clojure.land/projects.edn

```diff
@@ -5958,4 +5958,22 @@
   :url "https://github.com/kkinnear/zprint",
   :platforms #{"clj" "cljs"},
   :key :zprint,
-  :tags #{"Code Formatting Tools"}}]
+  :tags #{"Code Formatting Tools"}}
+ {:name "config",
+  :url "https://github.com/lambdaisland/config",
+  :platforms #{"clj"},
+  :description "Clojure configuration library",
```

Comment:
Better to leave out the `:description` key and we will pull it from the Github repo on each daily sync